test/install.bats

```diff
@@ -1,7 +1,68 @@
 #!/usr/bin/env bats
 
+
+
+setup() {
+  # Ensure a pristine homedir for tests.
+  HOME="$(mktemp -d bats-asdf-hashicorp-XXXXXXXXXXXXXXX)"
+  # So we don't have to do rm -rf "${HOME}".
+  TEMP_HOME="${HOME}"
+  asdf_gitdir="$(dirname "${BATS_TEST_FILENAME}")/../.git"
```

Comment:
I think it should be the repository root, not `.git` dir.

Reply:
GitHub is running the workflow with https://github.com/asdf-community/asdf-hashicorp/pull/29/files#diff-126bf89616b7daa3d14ebc882ad18666aaf1c3dae888c4ba306a66ec80758bc1L52 I believe, and it's installing the Terraform plugin before these tests can even run.

Ideally, I'd test this against the _newer_  version of that file. Am I missing something?

Reply:
I think if I move the plugin additions to there, this might "just work". Of course, it would require more local setup.

Fingers crossed.

Reply:
See this document, in this example, you are now specifying `/home/runner/work/my-repo-name/my-repo-name/.git`, not `/home/runner/work/my-repo-name/my-repo-name`.
https://docs.github.com/en/actions/reference/environment-variables

Reply:
Oops, disregard.